libs/translations-finder/README.md

```diff
@@ -0,0 +1,81 @@
+# Translations Finder
+Finds missing translations in your project.
+
+## Installation
+Install with npm
+```shell script
+# Locally
+npm i translations-finder
+
+# Globally
+npm i -g translations-finder
+```
+
+## Usage
+```typescript
+import { findMissingTranslations } from 'translations-finder';
+
+const missingTranslations = findMissingTranslations(
+    definedTranslations,
+    globFilePattern,
+    regexTransaltion,
```

Comment:
typo

Reply:
Should I declare these variables with types above the invocation of this function? Or maybe add them to the description which explains them?

![Screenshot from 2019-12-16 16-32-33](https://user-images.githubusercontent.com/49659466/70919877-ee2a9880-2021-11ea-8a12-c980fae84e6c.png)